.github/labeler.yml

```diff
@@ -85,3 +89,43 @@ windows/config/: windows/src/desktop/**
 windows/engine/: windows/src/engine/**
 # windows/resources/
 # windows/samples/
+
+#
+# The lower part of the file adds labels based on the PR title
+#
+
+labels:
+  feat:
+    - '^feat(\(|:)'
+  fix:
+    - '^fix(\(|:)'
+  chore:
+    - '^chore(\(|:)'
+  #    "change",
+  docs:
+    - '^docs(\(|:)'
+  #    "style",
+  refactor:
+    - '^refactor(\(|:)'
+  #    "test",
+  auto:
+    - '^auto(\(|:)'
+  # Below are the scopes that we look for in the PR title
+  'android/':
+    - '\(.*android.*\):'
+  'common/':
+    - '\(.*common.*\):'
+  'developer/':
+    - '\(.*developer.*\):'
+  'ios/':
+    - '\(.*ios.*\):'
+  'linux/':
+    - '\(.*linux.*\):'
+  'mac/':
+    - '\(.*mac.*\):'
+  'oem/':
+    - '\(.*oem.*\):'
+  'web/':
+    - '\(.*web.*\):'
+  'windows/':
+    - '\(.*windows.*\):'
```

Comment:
Generally the code changes check will find these scopes for us so this is probably not necessary.

Reply:
This is useful if a change is e.g. for Linux, but it happens in `common/core`, e.g. packaging related. Or a linux specific change under `resources` or `docs`.